scripts/olm-create-bundle.sh

```diff
@@ -10,14 +10,16 @@ ROOT_DIR="$SCRIPTS_DIR/.."
 BUILD_DIR="$ROOT_DIR/build"
 DEFAULT_BUNDLE_CHANNEL="alpha"
 PRESERVE=${PRESERVE:-"false"}
+DEFAULT_OPERATOR_SDK_BIN_PATH="$ROOT_DIR/../../aws-controllers-k8s/code-generator/bin/operator-sdk"
+OPERATOR_SDK_BIN_PATH=${OPERATOR_SDK_BIN_PATH:-$DEFAULT_OPERATOR_SDK_BIN_PATH}
```

Comment:
I'm thinking this script's Usage statement should be updated to indicate that `OPERATOR_SDK_BIN_PATH` is a configurable.

Reply:
Updated the `USAGE`